com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -498,13 +498,13 @@ private static File createAppBundleBaseZip(Project project, File outDir, File ap
 
             // copy extension and bundle resoources
             Map<String, IResource> bundleResources = ExtenderUtil.collectBundleResources(project, getArchitectures(project));
-            final String assetsPath = "assets" + File.separator;
-            final String libPath = "lib" + File.separator;
-            final String resPath = "res" + File.separator;
+            final String assetsPath = "assets/";
```

Comment:
I was being too clever. ExtenderUtil.collectBundleResources() always returns all filenames with forward slashes.

Reply:
Tested on Windows+macOS to cover both forward and backward slashes in paths